train.py

```diff
@@ -477,14 +479,28 @@ def main(args):
 		if args.verbose:
 			onnxruntime.set_default_logger_severity(0)
 		(logits_, ) = onnxruntime_session.run(None, dict(x = waveform_input.cpu().numpy()))
-		assert torch.allclose(logits.cpu(), torch.from_numpy(logits_), rtol = 1e-02, atol = 1e-03)
-		
-		#model_def = onnx.load(args.onnx)
-		#import onnx.tools.net_drawer # import GetPydotGraph, GetOpNodeProducer
-		#pydot_graph = GetPydotGraph(model_def.graph, name=model_def.graph.name, rankdir="TB", node_producer=GetOpNodeProducer("docstring", color="yellow", fillcolor="yellow", style="filled"))
-		#pydot_graph.write_dot("pipeline_transpose2x.dot")
-		#os.system('dot -O -Gdpi=300 -Tpng pipeline_transpose2x.dot')
-		# add metadata to model
+
+		assert torch.allclose(
+				logits.cpu(),
+				torch.from_numpy(logits_),
+				**{'rtol': 1e-01, 'atol': 1e-02} if args.fp16 else {'rtol': 1e-02, 'atol': 1e-03}
+		)
+
+		if args.onnx_dot_file:
+			# Previosly install graphvis with: sudo apt-get install graphviz; pip install onnx onnxruntime-gpu pydot
```

Comment:
Typo: graphvis -> graphviz. Could you please move installation instructions to README + scripts/Dockerfile